openenade.als

```diff
@@ -0,0 +1,82 @@
+module openEnade
+
+sig CodigoPostal, CodigoIES, CodigoCurso, CodigoArea, NomeCurso, NomeUniversidade, Ano {}
+
+sig Regiao {
+	estados : some Estado,
+	universidades: set UniversidadePrivada
```

Comment:
Is necessary to have bidirectional relationships? I think that the diagram has only unidirectional relations.